app/root.tsx

```diff
@@ -14,7 +16,17 @@ import { SiteWrapper } from '~/components/site-wrapper/site-wrapper';
 import { ROUTES } from '~/router/config';
 import '~/styles/index.scss';
 
+export async function loader() {
+    return json({
+        ENV: {
+            VITE_WIX_CLIENT_ID: globalThis?.process?.env?.VITE_WIX_CLIENT_ID,
```

Comment:
`VITE` prefix was added for the Vite magic.. we can remove it now

Reply:
but in the instruction it is said to use process.env:
![image](https://github.com/user-attachments/assets/7e835664-3297-4c78-9c2b-f9b39f28711d)

Reply:
why do you need globalThis ?

Reply:
remove VITE_ prefix

Reply:
@VladyslavG , you are right, I don't need globalThis... however, I need question marks, cause in case of error on server side it is crashing otherwise because can't read process.env (not sure why)